http/headers/upgrade-insecure-requests.json

```diff
@@ -15,8 +15,7 @@
               "version_added": "44"
             },
             "edge": {
-              "version_added": false,
-              "notes": "<a href='https://developer.microsoft.com/en-us/microsoft-edge/platform/status/cspupgradeinsecurerequestsdirective'>Under consideration</a> for future release."
+              "version_added": "17",
```

Comment:
Please remove the [trailing comma](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) (not allowed in JSON :confused:).